scripts/dts/gen_defines.py

```diff
@@ -653,6 +653,8 @@ def write_vanilla_props(node):
                     subval_as_token = edtlib.str_as_token(subval)
                     macro2val[macro + f"_IDX_{i}_TOKEN"] = subval_as_token
                     macro2val[macro + f"_IDX_{i}_UPPER_TOKEN"] = subval_as_token.upper()
+                    macro2val[macro + f"_IDX_{i}_STRING_TOKEN"] = subval_as_token
```

Comment:
we shouldn't have both `_TOKEN` and `_STRING_TOKEN` that are identical.

Reply:
I just left it as I did not want to be rude and remove somebody's work. But we had already discussed it with @mbolivar-nordic offline. And it seems that the one that was here sneaked through review in some totally different PR. It is hard for me to follow it now, as the whole discussion is on other machine. I mentioned that here: https://github.com/zephyrproject-rtos/zephyr/pull/46249#discussion_r930419980